multicast/send.py

```
@@ -341,10 +341,10 @@ def _sayStep(group, port, data):
 		_success = False
 		sock = multicast.genSocket()
 		if __debug__:
-				module_logger.info(
-					"Preparing to send %d",  # lazy formatting to avoid PYL-W1203
-					len(data),
-				)
+			module_logger.info(
```

Flake8 (7.2.0)

[warning] indentation contains tabs

(W191)